### ISSUE TITLE
github convert doesn't return valid steps

### ISSUE THREAD

**Issue by rwc9u** [harness/harness-migrate#105]
As an example the following step from a github workflow:

```yaml
      - name: Debug BuildKit
        if: always()
        run: |
          docker exec $(docker container ls | grep buildx | awk '{ print $(NF) }' | head -1) buildctl debug workers -v
          echo "-------> docker buildx du --verbose"
          docker buildx du --verbose
          echo "-------> docker system df"
          docker system df
          echo "-------> docker container ls"
          docker container ls
          echo "-------> docker image ls"
          docker image ls
          echo "-------> docker volume ls"
          docker volume ls
          echo "-------> df"
          df -h
```
The convert emits 

```yaml
   - name: Debug BuildKit
      spec:
        run: |
          docker exec $(docker container ls | grep buildx | awk '{ print $(NF) }' | head -1) buildctl debug workers -v
          echo "-------> docker buildx du --verbose"
          docker buildx du --verbose
          echo "-------> docker system df"
          docker system df
          echo "-------> docker container ls"
          docker container ls
          echo "-------> docker image ls"
          docker image ls
          echo "-------> docker volume ls"
          docker volume ls
          echo "-------> df"
          df -h
      type: script
```

This doesn't seem to be a valid yaml syntax for the next gen pipeline. I had to change to the following:

```yaml
              - step:
                  identifier: debug_buildkit
                  name: Debug BuildKit
                  type: Run
                  spec:
                    shell: Bash
                    command: |
                      docker exec $(docker container ls | grep buildx | awk '{ print $(NF) }' | head -1) buildctl debug workers -v
                      echo "-------> docker buildx du --verbose"
                      docker buildx du --verbose
                      echo "-------> docker system df"
                      docker system df
                      echo "-------> docker container ls"
                      docker container ls
                      echo "-------> docker image ls"
                      docker image ls
                      echo "-------> docker volume ls"
                      docker volume ls
                      echo "-------> df"
                      df -h
```

**Comment by jimsheldon** [comment]
Thanks for opening an issue @rwc9u 

The first conversion you see is a newer yaml format that is in development, but not yet released.

You can pass the `--downgrade` flag to generate the yaml you are looking for.

For example:
```
harness-migrate github convert --downgrade action.yml
```

**Comment by jimsheldon** [comment]
Fixed in #106, `--downgrade` will be the default behavior in the next release.
